bin/pp_ast.ml

```diff
@@ -0,0 +1,122 @@
+open Ppxlib
+
+module Kind = struct
+  type t = Signature | Structure | Expression | Pattern | Core_type
+
+  let to_utils_kind = function
+    | Structure -> Ppxlib__Utils.Kind.Impl
```

Comment:
Do you think Utils should be properly exposed behind a `Ppxlib.Private` module?

Reply:
Yeah that might be a good idea!